qutip/installsettings.py

```diff
@@ -58,18 +58,12 @@ class InstallSettings:
         _logger = None
 
     try:
-        qutip_conf_dir = os.path.join(os.path.expanduser("~"), '.qutip')
-        if not os.path.exists(qutip_conf_dir):
-            os.mkdir(qutip_conf_dir)
-        tmproot = os.path.join(qutip_conf_dir, 'coeffs')
+        tmproot = os.path.join(os.path.expanduser("~"), '.qutip')
```

Comment:
This is safe on Windows, right?

Reply:
https://docs.python.org/3/library/os.path.html#os.path.expanduser is documented as supporting Windows.